package.json

```diff
@@ -37,7 +37,7 @@
     "semver": "^5.5.1"
   },
   "devDependencies": {
-    "@bpanel/bpanel-ui": "0.0.13",
+    "@bpanel/bpanel-ui": "^0.0.14",
```

Comment:
should probably increase in peerDeps too

Reply:
Updated